.github/workflows/optimize-health-status-report.yml

```diff
@@ -48,7 +48,8 @@ jobs:
       - name: Send report to slack channel
         uses: slackapi/slack-github-action@485a9d42d3a73031f12ec201c457e2162c45d02d # v2.0.0
         with:
-          channel-id: "#optimize-status"
-          slack-message: ${{ steps.get-report.outputs.infra_report }}
-        env:
-          SLACK_BOT_TOKEN: ${{ steps.secrets.outputs.SLACK_API_KEY }}
+          method: chat.postMessage
```

Comment:
see https://github.com/slackapi/slack-github-action/releases/tag/v2.0.0#the-slack-api-method-now-must-be-specified-in-inputs